test/extended/baremetal/hosts.go

```diff
@@ -187,3 +188,37 @@ var _ = g.Describe("[sig-installer][Feature:baremetal][Serial] Baremetal platfor
 		o.Expect(found).To(o.BeFalse())
 	})
 })
+
+var _ = g.Describe("[sig-installer][Feature:baremetal] A baremetal deployment without a provisioning network should", func() {
```

Comment:
Maybe we can move this under `baremetal platform should` tests as `g.It`?

Reply:
Wouldn't be safer to move this test under the `serial` suite? If the update action will have success - by mistake - it could create some disruption for the subsequent baremetal tests

Reply:
this is probably a good idea

Reply:
should I move the entire test block inside the block at L154 or just add the Serial notation here ?

Reply:
That block is mostly required in case your test would need an extra worker to work with, otherwise it seems fine to just add the serial tag (it will save some time by avoiding to deploy the extra worker)